pvlib/pvarray.py

```diff
@@ -274,6 +305,11 @@ def huld(effective_irradiance, temp_mod, pdc0, k=None, cell_type=None):
     cell_type : str, optional
         If provided, must be one of ``'cSi'``, ``'CIS'``, or ``'CdTe'``.
         Used to look up default values for ``k`` if ``k`` is not specified.
+    k_version : str, optional
+        Either ``'pvgis5'`` (default) or ``'pvgis6'``. Selects values
+        for ``k`` if ``k`` is not specified. If ``'pvgis5'``, values are
+        from PVGIS documentation and are labeled in [2]_ as "current".
+        If ``'pvgis6'`` values are from [2]_ labeled as "updated".
```

Comment:
```suggestion
        Selects values for ``k`` if ``k`` is not specified: either ``'pvgis5'`` (default)
        or ``'pvgis6'``.  For ``'pvgis5'``, ``k`` values are from PVGIS documentation, 
        which are also identified as "current" in [2]_ . For ``'pvgis6'``, values are 
        from [2]_ where they are labeled as "updated".
```

Reply:
I'll not take this suggestion as the phrasing implies that a value of `'pvgis5'` is assigned to `k` rather than being a value of `k_version`.